uint64 inc
uint8 followerSX
uint8 followerSXC
uint8 followerDXC
uint8 followerDX